cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/StopStartUpscaleStartInstancesHandler.java

```diff
@@ -165,7 +165,8 @@ private List<CloudVmInstanceStatus> getInstanceStatusOnStartError(CloudConnector
     private List<CloudVmInstanceStatus> collectStoppedInstancesFromCloudProvider(
             CloudConnector connector, AuthenticatedContext ac, List<CloudInstance> cloudInstances) {
         List<CloudVmInstanceStatus> vmInstanceStatusAllInstances = connector.instances().checkWithoutRetry(ac, cloudInstances);
-        return vmInstanceStatusAllInstances.stream().filter(vm -> vm.getStatus() == InstanceStatus.STOPPED).collect(Collectors.toList());
+        return vmInstanceStatusAllInstances.stream().filter(vm -> (vm.getStatus() == InstanceStatus.STOPPED) || (vm.getStatus() == InstanceStatus.PENDING))
+                .collect(Collectors.toList());
     }
```

Comment:
Re-adding my earlier comment from [here](https://github.com/hortonworks/cloudbreak/pull/14758#discussion_r1193544755):

> Could you please cover the new behavior with an UT?

I see no occurrence of "PENDING" in the most recent [StopStartUpscaleStartInstancesHandlerTest](https://github.com/hortonworks/cloudbreak/blob/b3c1695616805724e9e4e7272fdf49fb5d260d70/cloud-reactor/src/test/java/com/sequenceiq/cloudbreak/cloud/handler/StopStartUpscaleStartInstancesHandlerTest.java). Actually, the purpose of this method is to extract all the instances that are stopped or pending, so perhaps the name could be also adapted accordingly.